cabot/urls.py

```diff
@@ -175,31 +175,13 @@ def append_plugin_urls():
         try:
             _module = import_module('%s.urls' % plugin)
         except Exception as e:
-            pass
+            if not e.message.startswith('No module named'):
```

Comment:
That feels a bit hackish ?

Reply:
hmm...yeah i think actually just catching ImportError should work